### PR TITLE
Search modal enhancements

### PR DESCRIPTION
**Pull request by patricksptang**
Enhanced the search modal in the display of search suggestions of:
  - DStrong words
  - Original language words with LexicalGroup

Remove translated messages which are no longer used.

Update the translated message with the latest in Crowdin

Added the "Advanced search" mode in the search modal

### CODE REVIEW

File: step-core-data/src/main/resources/com/tyndalehouse/step/core/data/create/lexicon/lexicon_greek.txt [hunk 1]
```diff
@@ -62370,7 +62370,7 @@ $=G2424I=ιησους==============
 @StrBetaAccented=	*)ihsou=s
 @StrTranslit=	Iēsoûs
 @StrPronunc=	ee-ay-sooce'
-@Translations=	Jesus/Joshua
+@Translations=	Joshua
```
Comment (danbenn) [inline step-core-data/src/main/resources/com/tyndalehouse/step/core/data/create/lexicon/lexicon_greek.txt:62373]: Do we want the "translations" field to only be one word?

File: step-web/src/main/webapp/js/step.util.js [hunk 2]
```diff
@@ -399,10 +399,13 @@ step.util = {
      */
     createNewLinkedColumn: function (passageId) {
 		if ($(window).width() < 768) {
-			var msg = "Your screen is not wide enough to open another panel.";
+			if (step.util.localStorageGetItem("already_warned_screen_not_wide_enough") !== "true") {
+				var msg = __s.screen_not_wide_enought;
 			if ((step.touchDevice) && ($(window).height() > 768))
-				msg += " Rotate your screen to horizontal mode if available.";
+					msg += " " + __s.rotate_screen_to_landscape_mode;
```
Comment (danbenn) [inline step-web/src/main/webapp/js/step.util.js:405]: thanks for adding this, would be good to get this in crowdin too!

**Review by danbenn — COMMENTED** [review]
I don't fully understand all the changes but left a small amount of feedback! Excited for the search updates!

File: step-core-data/src/main/resources/com/tyndalehouse/step/core/data/create/lexicon/lexicon_greek.txt (hunk 1 above)
Comment (patricksptang) [reply step-core-data/src/main/resources/com/tyndalehouse/step/core/data/create/lexicon/lexicon_greek.txt:62373]: David and another person will make the changes for many other words.

File: step-web/src/main/webapp/js/step.util.js (hunk 2 above)
Comment (patricksptang) [reply step-web/src/main/webapp/js/step.util.js:405]: Agree

**Pull request merged**